.docker-compose-test-delegate.env

```diff
@@ -0,0 +1,18 @@
+CANTALOUPE_ENDPOINT_ADMIN_SECRET=localpassword
+CANTALOUPE_ENDPOINT_ADMIN_ENABLED=true
+
+# TODO: fill these in
+# $CANTALOUPE_SOURCE_STATIC=
+
+cantaloupe_endpoint_admin_secret=secret
+CANTALOUPE_ENDPOINT_ADMIN_ENABLED=true
+CANTALOUPE_DELEGATE_SCRIPT_ENABLED=true
+CANTALOUPE_DELEGATE_SCRIPT_PATHNAME=/usr/local/cantaloupe/delegates.rb
+CANTALOUPE_DELEGATE_SCRIPT_CACHE_ENABLED=false
+CANTALOUPE_SOURCE_DELEGATE=false
+CANTALOUPE_S3SOURCE_BASICLOOKUPSTRATEGY_PATH_PREFIX=.jpx
+CANTALOUPE_S3SOURCE_BASICLOOKUPSTRATEGY_BUCKET_NAME=bucketeer-test-data
```

Comment:
Maybe say "getyourown" here too to make it clear they'll need their own bucket... ours won't work for them.

Reply:
You decided to not do this one or missed it?